esbuild.config.mjs

```diff
@@ -44,7 +44,7 @@ const context = await esbuild.context({
 
 	// We don't need to include code to create zip files (deflate), only read them (inflate),
 	// so this cuts it out and makes the final bundle smaller.
-	alias: {'@zip.js/zip.js': '@zip.js/zip.js/lib/zip-no-worker-inflate.js'},
+	// alias: {'@zip.js/zip.js': '@zip.js/zip.js/lib/zip-no-worker-inflate.js'},
```

Comment:
This seems to be the issue, more specifically it's related to the workers. 

I'm not sure entirely what is causing this issue, but it seems like whenever the code reaches anything related to `zip.js`, it attempts to fetch it via web worker?

Reply:
Hmm but that's weird, I'm specifically mapping it to `zip-no-worker-inflate` which shouldn't be using workers at all. Also in my testing I couldn't reproduce the issue; I was able to list files and get their contents from the zip.

Reply:
Alright, well it doesn't block me for development as I am able to get around it. This ticket should be ready to go.